docs/sorbet.md

```diff
@@ -6,6 +6,19 @@ Mutant aims to implement full sorbet support. Full support includes:
 * Being able to cover mutations via the type system. This capability is being
   developed right now.
 
+For now, all of the mutations to `T` methods are skipped. To better leverage `mutant`'s capabilities
```

Comment:
idk if this is the best place, placement, or phrasing, but it's an honest attempt.